<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Path to file (.inp) or folder</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="vecInputFileEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QToolButton" name="vecInputFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="vecVecFileEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Path to VEC file</string>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QToolButton" name="methodOutputFolderButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Path to output folder</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QToolButton" name="vecOutputFolderButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Path to new method file</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLineEdit" name="methodMethodFileEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="18" column="0" colspan="3">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLineEdit" name="methodOutputFolderEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="19" column="0" colspan="3">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Molecular Delimiter</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="20" column="2">
      <widget class="QToolButton" name="delimiterXyzFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Path to output folder</string>
       </property>
      </widget>
     </item>
     <item row="21" column="1">
      <widget class="QLineEdit" name="delimiterOutputFolderEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="21" column="2">
      <widget class="QToolButton" name="delimiterOutputFolderButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="22" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Minimum Atoms Per Molecule</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QToolButton" name="vecVecFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="vecOutputFolderEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="3">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Change Method group (header)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Path to file (.inp) or folder</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLineEdit" name="methodInputFileEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="vecChangeVecButton">
       <property name="text">
        <string>Change $VEC</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QToolButton" name="methodInputFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QToolButton" name="methodMethodFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Path to output folder</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QPushButton" name="methodChangeMethodButton">
       <property name="text">
        <string>Change Method</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Path to initial file (.xyz)</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QLineEdit" name="delimiterXyzFileEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Interpolator</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Path to initial file (.xyz)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="interpolationInitialFileEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="interpolationInitialFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="interpolatorOpenFolderButton">
       <property name="text">
        <string>Open output folder</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="interpolationTransformedFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Change $VEC group</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="interpolationOutputFolderEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="interpolationRunButton">
       <property name="text">
        <string>Interpolate</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="interpolationTransformedFileEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="interpolationOutputFolderButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Path to transformed file (.xyz)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Amount of steps</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Path to output folder</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="interpolationStepsAmountBox">
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="23" column="1">
      <widget class="QSpinBox" name="delimiterBondAtomsBox">
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="24" column="0">
      <widget class="QPushButton" name="delimiterSplitButton">
       <property name="text">
        <string>Split</string>
       </property>
      </widget>
     </item>
     <item row="23" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Molecular Bond Limit</string>
       </property>
      </widget>
     </item>
     <item row="25" column="0" colspan="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="22" column="1">
      <widget class="QSpinBox" name="delimiterMinAtomsBox">
       <property name="maximumSize">
        <size>
         <width>366</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="27" column="0">
      <widget class="QPushButton" name="plotRunEditorButton">
       <property name="text">
        <string>Plot Editor</string>
       </property>
      </widget>
     </item>
     <item row="26" column="0" colspan="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Plot results</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="helpMenu"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
